manifests/klusterlet/managed/klusterlet-registration-clusterrole.yaml

```diff
@@ -10,6 +10,14 @@ rules:
 - apiGroups: [""]
   resources: ["nodes"]
   verbs: ["get", "list", "watch"]
+# get pods and replicasets is for event creation
```

Comment:
I do not think we need permission on managed cluster, we only need it in management part, right?

Reply:
the klusterlet registration/work agents also create event on managed cluster, so need the permission too

Reply:
removed since added on the wrong place.